perun-base/src/test/resources/test-schema.sql

```diff
@@ -1638,6 +1638,15 @@ create table consent_attr_defs (
 	constraint consentatt_consent_fk foreign key (consent_id) references consents(id)
 );
 
+-- CREATED_APP_NOTIFICATIONS_SENT - applications for which notification APP_CREATED_VO_ADMIN was sent
+create table created_app_notifications_sent (
+	app_id int not null,
+	created_at timestamp default statement_timestamp() not null,
```

Comment:
What about making it more generic so it could be extended in the future by other types of notifications?